jujugui/static/gui/src/app/utils/bakery-utils.js

```diff
@@ -1,28 +1,13 @@
-/*
-This file is part of the Juju GUI, which lets users view and manage Juju
-environments within a graphical interface (https://launchpad.net/juju-gui).
-Copyright (C) 2017 Canonical Ltd.
-
-This program is free software: you can redistribute it and/or modify it under
-the terms of the GNU Affero General Public License version 3, as published by
-the Free Software Foundation.
-
-This program is distributed in the hope that it will be useful, but WITHOUT
-ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
-SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
-General Public License for more details.
-
-You should have received a copy of the GNU Affero General Public License along
-with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
-
+/* Copyright (C) 2017 Canonical Ltd. */
 'use strict';
 
 /**
   Helper functions for creating and working with the bakery.
 */
 
 const Notification = require('../components/notification/notification');
+const React = require('react');
+const ReactDOM = require('react-dom');
```

Comment:
Just a driveby looks like they were missed in @Lukewh 's branch which I had to land to continue on this...nbd!